DEVELOPMENT.md

```diff
@@ -0,0 +1,29 @@
+# DEVELOPMENT NOTES
```

Comment:
This whole file should probably be turned into a `CONTRIBUTING.md` rather than `DEVELOPMENT.md`